content/concepts/transports/webrtc.md

```diff
@@ -1,9 +1,142 @@
 ---
 title: "WebRTC"
-description: "WebRTC is a new specification that uses QUIC to offer an alternative to WebSocket. Conceptually, it can be considered WebSocket over QUIC.Learn about WebTransport and how it is used in libp2p."
+description: "WebRTC is a protocol for real-time communication and is used to establish connections between browsers and other nodes."
 weight: 110
 aliases:
     - "/concepts/transports/webrtc"
 ---
 
-Coming soon!
+## What is WebRTC?
+
+[WebRTC (Web Real-Time Communications)](https://webrtc.org/) is a framework for real-time communication
+and is used to establish browser-to-server and browser-to-browser connections between applications.
+WebRTC serves as a good choice for applications that need built-in support for media
+communication and do not have specific requirements for the underlying transport
+protocol.
+
+### Key features for libp2p
+
+
+- Peer connections: WebRTC enables [direct peer-to-peer connections](https://webrtc.org/getting-started/peer-connections)
+  between browsers and other nodes through a process called [signaling](https://webrtc.org/getting-started/peer-connections#signaling).
+  In non-decentralized WebRTC, this is facilitated by a [TURN server.](https://webrtc.org/getting-started/turn-server)
+  This allows for faster and more efficient communication. Peers can also retrieve or consume the media and
+  also produce it.
+
+- Data channel: WebRTC also provides peer-to-peer data channels called
+  [WebRTC data channels](https://developer.mozilla.org/en-US/docs/Games/Techniques/WebRTC_data_channels),
+  which works on
+  [SCTP (Stream Control Transmission Protocol)](https://en.wikipedia.org/wiki/Stream_Control_Transmission_Protocol).
+  A WebRTC data channel allows applications to send text or binary data over an active connection to a peer.
+  Data channels can be [ordered or unordered](https://developer.mozilla.org/en-US/docs/Web/API/RTCDataChannel/ordered),
+  where ordered channels guarantee the message delivery order (like running on TCP), and unordered channels increase
+  speed but deliver messages out of order (like UDP.)
```

Comment:
I don't think this piece of information is particularly helpful to the user, especially since libp2p only uses ordered data channels. I'd remove this sentence.